lib/game_team.rb

```diff
@@ -1,6 +1,10 @@
 class GameTeam
+  attr_reader :team_id, :goals
+
   def initialize(details)
     @game_id = details[:game_id]
+    @team_id = details[:team_id].to_i
+    @goals = details[:goals].to_i
```

Comment:
This might be better converted to a symbol rather than a string. Let's debug this later